dashboard/v1.0/package-lock.json

```diff
@@ -12899,34 +12894,6 @@
       "resolved": "https://registry.npmjs.org/react-lifecycles-compat/-/react-lifecycles-compat-3.0.4.tgz",
       "integrity": "sha512-fBASbA6LnOU9dOU2eW7aQ8xmYBSXUIWr+UmF9b1efZBazGNO+rcXT/icdKnYm2pTwcRylVUYwW7H1PHfLekVzA=="
     },
-    "react-motion": {
-      "version": "0.4.8",
-      "resolved": "https://registry.npmjs.org/react-motion/-/react-motion-0.4.8.tgz",
-      "integrity": "sha1-I7st0nwtjgDSKeRVctEF789Ao14=",
-      "requires": {
-        "create-react-class": "^15.5.2",
-        "performance-now": "^0.2.0",
-        "prop-types": "^15.5.8",
-        "raf": "^3.1.0"
-      },
-      "dependencies": {
-        "performance-now": {
-          "version": "0.2.0",
-          "resolved": "https://registry.npmjs.org/performance-now/-/performance-now-0.2.0.tgz",
-          "integrity": "sha1-M+8wxcd9TqIcWlOGnZG1bY8lVeU="
-        }
-      }
-    },
-    "react-motion-ui-pack": {
-      "version": "0.10.3",
-      "resolved": "https://registry.npmjs.org/react-motion-ui-pack/-/react-motion-ui-pack-0.10.3.tgz",
-      "integrity": "sha512-Pb92CWNjYovx/W318D4mQPrkw/nBG1eixi7f8jV4qaHitXMh1URCgwDS9XnuJNqubv6cyVfEzMzScEFrO2/WAQ==",
-      "requires": {
-        "get-prefix": "^1.0.0",
-        "prop-types": "^15.5.10",
-        "react-motion": "^0.4.5"
-      }
-    },
```

Comment:
Same here.

Reply:
Reverted the package-lock-json changes @muskankhedia .